src/tactic/core.lean

```diff
@@ -59,6 +59,17 @@ meta def mfoldl {α : Type} {m} [monad m] (f : α → expr → m α) : α → ex
 | x e := prod.snd <$> (state_t.run (e.traverse $ λ e',
     (get >>= monad_lift ∘ flip f e' >>= put) $> e') x : m _)
 
+meta def mk_bin_op_expr (op ident : expr) : list expr → expr
```

Comment:
duplicate of `mk_op_lst` in `expr`. The next 2 declarations are also duplicates.